com.github.louis77.tuner.yml

```diff
@@ -0,0 +1,56 @@
+---
+app-id: com.github.louis77.tuner
+runtime: io.elementary.Platform
+runtime-version: '6.1'
+sdk: io.elementary.Sdk
```

Comment:
this runtime isn't available on Flathub.

Reply:
Hubert,
Is there a way to get that runtime into the buildbot at buildtime? Can the app be built locally and uploaded to flathub, bypassing the buildbot?

Reply:
short answer no.

longer answer, I suggested it be submitted, but the idea didn't get traction.

There is a base app for "Juno" but I'm not sure if that's a recent enough Elementary platform. Otherwise it's my understanding that the Elementary runtime is just the GNOME runtime + a few packages, so that could be approach taken.

Reply:
How about that "a few packages" be provided through shared-modules to make conversion to what Flathub supports more obvious?

Reply:
What I would strongly suggest here is to ask if the elementary developers are fine publishing their runtime here, duplicating the work when we already have too many things on our plate is not ideal.

Reply:
I strongly disagree. Same runtime should not have multiple sources, that will break.

Reply:
If this really needs Elementary SDK rather than gnome, it should be in Elementary repo.